version/version.go

```diff
@@ -17,9 +17,6 @@ func init() {
 const (
 	// TMCoreSemVer is the current version of Tendermint Core.
 	// It's the Semantic Version of the software.
-	// Must be a string because scripts like dist.sh read this file.
-	// XXX: Don't change the name of this variable or you will break
-	// automation :)
 	TMCoreSemVer = "0.34.0"
```

Comment:
This was used for the previous release system. We could remove it and go with: `git describe --always) | sed 's/^v//'`. Is there a preference to keep it this was?

Reply:
If you use git annotated tags, you could just use `git describe`

Reply:
We have started using annotated tags, but they are a bit different. For example the tag for master is ` v0.34.0-rc4-72-gd4f906609`. Do you happen to know why it gets tagged like this?

Reply:
Yep - I'm now sitting on top of master:

```
alessio@phoenix:~/work/tendermint$ git describe 
v0.34.0-rc4-72-gd4f906609
```

`git describe` takes the most recent tag reachable from a commit (`HEAD` in this case) and append suffixes to the tag name `with the number of additional commits on top of the tagged object and the abbreviated object name of the most recent commit`. So:

* **0.34.0-rc4** is the most recent tag reachable from `HEAD`.
* **72** is the count of commits since *0.34.0-rc4*.
* **d4f906609** is the latest commit.

`go` modules versioning works in an extremely similar way.

Reply:
I removed gitcommit so we don't show duplicate commit hashes

Reply:
Wise choice :)